light-client/tests/model_based.rs

```diff
@@ -446,18 +447,33 @@ fn single_step_test(
     let trusting_period: Duration = tc.initial.trusting_period.into();
     for (i, input) in tc.input.iter().enumerate() {
         output_env.logln(&format!("    > step {}, expecting {:?}", i, input.verdict));
+
+        // ------------------->
+        // Below is a temporary work around to get rid of bug-gy validator sorting
```

Comment:
Is there an issue for this sorting bug? Could you point us to any specific line where the bug occurs or you don't know?

Reply:
No, but yes there should be one. I think the problem is somewhere with ser/de, it somehow changes the sequence of validators in the set which then produces a wrong valset hash. I'm not sure how this is happening. But I'm sure this is not coming from Testgen at least. Currently, these tests were disabled on master because of this and re-sorting seemed to be a quick solution. I need to dig further to know where this problem is coming from.

Reply:
Issue for this: #687

Reply:
Ok, I think I know what is happening here. When we transform tests from Apalache counterexamples, the validators are in no particular order. At the same time new deserialization requires them to be sorted according to the power, and then lexicographically. Therefore the validation fails. One more point for implementing #665.